<launch>
    <!--Starting project16_coordinator-->
    <node pkg="project16_coordinator" type="tams_pr16_drink_chooser" name="tams_pr16_drink_chooser" output="screen">
        <rosparam command="load" file="$(find project16_coordinator)/config/cocktails.yaml" />
    </node>
    
</launch>
